gef.py

```diff
@@ -6445,16 +6468,19 @@ def __init__(self):
         return
 
     @only_if_gdb_running
-    @parse_arguments({}, {("--location", "-l"): 0, ("--show-opcodes", "-s"): True, "--length": 0})
+    @parse_arguments({("LOCATION"): "$pc"}, {("--show-opcodes", "-s"): True, "--length": 0})
```

Comment:
Why uppercase ? We usually reserve uppercases for globals

Reply:
Sorry, I am used to commandline interfaces showing usage instructions in the help message in the format of `--option OPTION`. I will fix it.